notebooks/slicing_tutorial_WiC.ipynb

```diff
@@ -0,0 +1,864 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snorkel-Workshop: Slicing Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "To start, let's make sure that we have the right paths/environment variables set by following the instructions in `snorkel-superglue/README.md`.\n",
+    "\n",
+    "Specifically, ensure that:\n",
+    "* `snorkel` is installed\n",
+    "* `PYTHONPATH` includes root of `snorkel-superglue`\n",
+    "* `SUPERGLUEDATA` is set where [download_superglue_data.py](https://github.com/HazyResearch/snorkel-superglue/blob/staging/download_superglue_data.py) was called."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys, os\n",
+    "import pandas as pd\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from snorkel.mtl.data import MultitaskDataLoader\n",
+    "from snorkel.mtl.model import MultitaskModel\n",
+    "from snorkel.mtl.snorkel_config import default_config as config\n",
+    "from snorkel.mtl.trainer import Trainer\n",
+    "from snorkel.slicing.sf import slicing_function\n",
+    "from snorkel.slicing.apply import PandasSFApplier\n",
+    "from snorkel.slicing.utils import add_slice_labels, convert_to_slice_tasks\n",
+    "from snorkel.types import DataPoint\n",
+    "\n",
+    "if not \"cwd\" in globals():\n",
+    "    cwd = Path(os.getcwd())\n",
+    "sys.path.insert(0, str(cwd.parents[0]))\n",
+    "\n",
+    "import models\n",
+    "from dataloaders import get_dataset, get_dataloaders\n",
+    "from tokenizer import get_tokenizer\n",
+    "from utils import task_dataset_to_dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll be working with the [Words in Context (WiC) task](https://pilehvar.github.io/wic/). To start, let's look at a few examples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explore the WiC dataset\n",
+    "Let's take a look at a few examples! "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['SUPERGLUEDATA'] = '/home/ubuntu/workspace/superglue_data/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "task_name = \"WiC\"\n",
+    "data_dir = os.environ[\"SUPERGLUEDATA\"]\n",
+    "max_sequence_length = 256\n",
+    "bert_model = \"bert-large-cased\"\n",
```

Comment:
If you're going to put this here (and maybe makes more sense to split up and go elsewhere), I'd add a few sentences introducing BERT.  In general: You'll give the detailed story in your slides / voiceover, but should include a minimal explanation in the notebook (so that someone going over it later could get what's going on)